AGGREGATE.md

```diff
@@ -329,6 +329,8 @@ encoded public keys is as follows:
 }
 ```
 
+Note: The version in the `.well-known` path may change.
```

Comment:
Maybe let's add a bit more color here, i.e. when this may change. (Something like "may be updated in future versions of the spec if the details of public key serving details change, especially in a backwards incompatible way".)

Reply:
Thanks, updated with the details.